src/Test/L0/Plugin/TestResultParser/PipelineTestRunPublisherL0.cs

```diff
@@ -0,0 +1,164 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Threading;
+using System.Threading.Tasks;
+using Agent.Plugins.Log.TestResultParser.Contracts;
+using Agent.Plugins.Log.TestResultParser.Plugin;
+using Agent.Plugins.TestResultParser.Plugin;
+using Microsoft.TeamFoundation.TestManagement.WebApi;
+using Microsoft.VisualStudio.Services.Common;
+using Moq;
+using Xunit;
+using TestOutcome = Agent.Plugins.Log.TestResultParser.Contracts.TestOutcome;
+using TestRun = Agent.Plugins.Log.TestResultParser.Contracts.TestRun;
+
+namespace Test.L0.Plugin.TestResultParser
+{
+    public class PipelineTestRunPublisherL0
+    {
+        [Fact]
```

Comment:
can we use mstest? :p

Reply:
Also please check for code coverage locally

Reply:
no :)

Reply:
We should atleast start usage with our tests. It'll give a choice for folks going ahead instead of following whats there

Reply:
@TingluoHuang any objections to using mstest v2 framework for these tests?

Reply:
@ShreyasRmsft we better have a single test framework with in this repository. any reason we have to use mstest instead of xunit?